<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScriptProgressDialog</class>
 <widget class="QDialog" name="ScriptProgressDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>94</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>360</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MFC Convert Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../gui/gui.qrc">
    <normaloff>:/gui/icons/knut-16.png</normaloff>:/gui/icons/knut-16.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="progressStatusLabel">
     <property name="text">
      <string>Progress status</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QProgressBar" name="conversionProgressBar">
     <property name="value">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Abort|QDialogButtonBox::Yes</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../gui/gui.qrc"/>
 </resources>
 <connections/>
</ui>
